server/controllers/chapterController.ts

```diff
@@ -21,8 +21,22 @@ export default {
   },
 
   async create(req: Request, res: Response) {
-    const { name, description, category, details } = req.body;
-    const chapter = new Chapter({ name, description, category, details });
+    const {
```

Comment:
This unfortunately wont work at property at runtime, beacuse here we implicitly convert location (string) from body into a Location object, change this to location_id and creator_id (for, now later we'll get that from logged in user), and query location and user before inserting a new chapter

Reply:
@Zeko369  i think at runtime TypeORM converts an id to an entity object

Reply:
Yeah, I was wrong, it actually work, but sending `location_id` and `creator_id` doesn't, so do you think we should change it to `location` and `creator` in `api docs` or make 2 more queries to the db here?

Reply:
@Zeko369 sorry for the delay. I think we should change it to `location` and `creator`. Since, TypeOrm fails if it is not a valid id we can choose to provide helpful error message instead of making 2 more request to the db.

Reply:
Yeah, let's do that then

Reply:
Alright cool, I am on it.